test/integration/controller/issuer/issuer_suite_test.go

```diff
@@ -47,19 +47,34 @@ var (
 	ctx context.Context
 	log logr.Logger
 
-	restConfig     *rest.Config
-	testEnv        *envtest.Environment
-	testClient     client.Client
-	kubeconfigFile string
+	restConfig           *rest.Config
+	testEnv              *envtest.Environment
+	testClient           client.Client
+	acmeDirectoryAddress string
```

Comment:
`acmeDirectoryAddress` has been added. This variable holds the directory address of the local Pebble ACME server that will be used to create `Issuer` resources in tests.
The test asserts that this variable has been set before the test starts.